graphql/util.go

```diff
@@ -427,46 +427,58 @@ func mapHTTPStatusToGqlError(ctx context.Context, httpStatus int, err error) *gq
 	}
 }
 
-func isTaskBlocked(ctx context.Context, at *restModel.APITask) (*bool, error) {
-	t, err := task.FindOneIdNewOrOld(*at.Id)
+func isTaskBlocked(ctx context.Context, at *restModel.APITask) (bool, error) {
```

Comment:
Both of these helper functions have to convert the API task document, and then just call one function. Is it possible to instead do the conversion in canRestartTask so we only need to do it once? Then we can just call t.Blocked() and t.IsPartOfDisplay() directly.